packages/instant-apps-components/CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v1.0.0-beta.191
+
```

Comment:
can you add a note of the change here

Reply:
Sure but it's the exact same as the note below it